Source/WebCore/platform/graphics/cairo/ImageBufferUtilitiesCairo.cpp

```diff
@@ -50,6 +50,17 @@
 
 namespace WebCore {
 
+static String imageTypeFromMIMEType(const String& mimeType)
+{
+    return mimeType.substring(sizeof "image");
+}
+
+bool platformMIMETypeIsJPEG(const String& mimeType)
+{
+    return imageTypeFromMIMEType(mimeType) == "jpeg"_s;
+
+}
```

Comment:
These function do not belong to this file. I think they should be moved to MIMETypeRegistry.cpp. See MIMETypeRegistry::isTextMIMEType() and isPostScriptMIMEType() as examples.

Reply:
Moved to MIMETypeRegistry.cpp as isJPEGMIMEType.